phpunit.xml

```diff
@@ -3,11 +3,26 @@
     colors="true"
     verbose="true"
 >
-    <!-- tests directories to run -->
-    <testsuite>
-        <directory>packages/*/tests</directory>
-        <directory>tests</directory>
-    </testsuite>
+    <testsuites>
+        <!-- only contrib open-source project Rectors -->
+        <testsuite name="contrib-rectors">
+            <directory>tests/Rector/Contrib</directory>
+        </testsuite>
+        <!-- only general configurable Rectors -->
+        <testsuite name="general-rectors">
+            <directory>tests/Rector/Dynamic</directory>
+            <directory>tests/Rector/MagicDisclosure</directory>
+        </testsuite>
+        <!-- and the rest -->
+        <testsuite name="main">
+            <directory>packages/*/tests</directory>
+            <directory>tests</directory>
+            <exclude>tests/Rector/Dynamic/</exclude>
+            <exclude>tests/Rector/MagicDisclosure/</exclude>
+            <exclude>tests/Rector/Contrib/</exclude>
+        </testsuite>
+    </testsuites>
+
```

Comment:
~20 % performance improvement

### Before

![image](https://user-images.githubusercontent.com/924196/34570084-0c1b21b2-f16b-11e7-8b2d-6c1180dfb73c.png)



### After 5 mins

![image](https://user-images.githubusercontent.com/924196/34570068-f7b25376-f16a-11e7-9dac-fb59dbde305b.png)

Reply:
Does it affects the `process` command?

Reply:
No, this is only related to Travis builds

Reply:
I've been thinking how should we improve it :thought_balloon: